### PR TITLE
Fix #3236: Hi-Fi Add Profile Tablet

### LINKED ISSUE #3236: Hi-Fi Add Profile Tablet
Mobile - https://xd.adobe.com/view/0687f00c-695b-437a-79a6-688e7f4f7552-70b6/screen/62215bbd-1f4e-448e-b370-19c4a06fa814/

Tablet - https://xd.adobe.com/view/d405de00-a871-4f0f-73a0-f8acef30349b-a234/screen/271b428d-7d98-40b4-b74e-f2a5266b0587/

**Follow this guide**: https://github.com/oppia/oppia-android/wiki/Working-on-UI

### PR DESCRIPTION
**Pull request by viktoriias**
## Explanation

Fixes #3236: Hi-Fi Add Profile Tablet:
- Remove `layout-land` file
- Introduce dimensions resources

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Comment by viktoriias** [comment]
**Portrait Phone**

Before             |  After
:-------------------------:|:-------------------------:
  |  
  |

**Comment by viktoriias** [comment]
**Landscape Phone**

Before             |  After
:-------------------------:|:-------------------------:
  |  
  |  
  |

**Comment by viktoriias** [comment]
@rt4914 Please review.

**Review by rt4914 — CHANGES_REQUESTED** [review]
@viktoriias The issue is related to two things:
1. Creating a single xml file instead of multiple files.
2. Creating Highfi for Tablet mocks too.

Currently this PR is doing 1st task only.

Also the approach mentioned in this PR has been to keep the margins fixed and the content width to be variable. But instead in general we are shifting towards keeping the content width fixed and margins to be variable. You can notice this in mocks too. For example: If you open [mobile-portrait](https://xd.adobe.com/view/0687f00c-695b-437a-79a6-688e7f4f7552-70b6/screen/62215bbd-1f4e-448e-b370-19c4a06fa814/specs/) and [mobile-landscape](https://xd.adobe.com/view/ee9e607b-dbd6-4372-48dc-b687d32af3da-98af/screen/645052fe-103b-4ca0-8250-e3b0d50bb169/specs/) mocks you will notice that maximum width of items is `304dp` which means that irrespective of screen width we are keeping it `304dp` width. And for [tablet-portrait](https://xd.adobe.com/view/d405de00-a871-4f0f-73a0-f8acef30349b-a234/screen/271b428d-7d98-40b4-b74e-f2a5266b0587) and [tablet-landscape](https://xd.adobe.com/view/d405de00-a871-4f0f-73a0-f8acef30349b-a234/screen/68765ce5-fc19-4340-b534-d43707f8e5c1/) this is `360dp` width.


So basically you just need to creating a container which holds all views in it correctly and based on which device/configuration it is you need to control the width of the container to be either `304dp` or `360dp`.

**Comment by viktoriias** [comment]
> @viktoriias The issue is related to two things:
> 
> 1. Creating a single xml file instead of multiple files.
> 2. Creating Highfi for Tablet mocks too.
> 
> Currently this PR is doing 1st task only.
> 
> Also the approach mentioned in this PR has been to keep the margins fixed and the content width to be variable. But instead in general we are shifting towards keeping the content width fixed and margins to be variable. You can notice this in mocks too. For example: If you open [mobile-portrait](https://xd.adobe.com/view/0687f00c-695b-437a-79a6-688e7f4f7552-70b6/screen/62215bbd-1f4e-448e-b370-19c4a06fa814/specs/) and [mobile-landscape](https://xd.adobe.com/view/ee9e607b-dbd6-4372-48dc-b687d32af3da-98af/screen/645052fe-103b-4ca0-8250-e3b0d50bb169/specs/) mocks you will notice that maximum width of items is `304dp` which means that irrespective of screen width we are keeping it `304dp` width. And for [tablet-portrait](https://xd.adobe.com/view/d405de00-a871-4f0f-73a0-f8acef30349b-a234/screen/271b428d-7d98-40b4-b74e-f2a5266b0587) and [tablet-landscape](https://xd.adobe.com/view/d405de00-a871-4f0f-73a0-f8acef30349b-a234/screen/68765ce5-fc19-4340-b534-d43707f8e5c1/) this is `360dp` width.
> 
> So basically you just need to creating a container which holds all views in it correctly and based on which device/configuration it is you need to control the width of the container to be either `304dp` or `360dp`.

I created a separate PR: https://github.com/oppia/oppia-android/pull/3724 - PTAL
